ersilia/setup/requirements/compound.py

```diff
@@ -11,7 +11,8 @@ def __init__(self):
             self.install()
 
     def install(self):
-        run_command("python -m pip install rdkit-pypi")
+        version = "2022.9.5"
```

Comment:
Where is this variable being used and it is also different from the version we are installing.

Reply:
@DhanshreeA Sorry I didn't noticed this variable. Now, it is fixed!